app/main/model/image.py

```diff
@@ -30,8 +32,30 @@ class ImageModel(db.Model):
   __table_args__ = (
     db.Index('ix_images_context', context, postgresql_using='gin'),
   )
+
+  @property
+  def canned_response(self):
+    model = app.config['IMAGE_MODEL']
+    if model and model.lower() == "pdq":
```

Comment:
could the model type keys be a constant somewhere so what we don't need to do text transformation for consistency?

Reply:
I'm doing these lowers to just prevent some potential casing error, but I'm going to remove it and let this fail noisily if it ever happens.